NOTICES.txt

```diff
@@ -1,586 +1,724 @@
-
-=============== TABLE OF CONTENTS =============================
-
-
-The following is a listing of the CyberArk Secrets Provider for K8s open source
-components detailed in this document. This list is provided for your convenience;
-please read further if you wish to review the copyright notice(s) and the full
-text of the license associated with each component.
-
-
-SECTION 1: Apache License 2.0
-
->>> github.com/cyberark/conjur-api-go v0.6.0
->>> github.com/cyberark/conjur-authn-k8s-client v0.16.1
->>> github.com/googleapis/gnostic v0.3.1
->>> github.com/modern-go/reflect2 v1.0.1
->>> gopkg.in/yaml.v2 v2.2.2

```

Comment:
why was `gopkg.in/yaml.v2` dropped? Don't we organize modules in alphabetical order?

Reply:
it's still there, but it wasn't in alphabetical order. will fix.